### PR TITLE
added verification of only one esdt transfer

### PR DESCRIPTION
**Pull request by sasurobert**
added verification of only one esdt transfer

### CODE REVIEW

**Review by catalinnnn — APPROVED** [review]

File: mock/context/runtimeContextWrapper.go [hunk 1]
```diff
@@ -247,14 +247,6 @@ func NewRuntimeContextWrapper(inputRuntimeContext *arwen.RuntimeContext) *Runtim
...
 	}
 
-	runtimeWrapper.IsWarmInstanceFunc = func() bool {
-		return runtimeWrapper.runtimeContext.IsWarmInstance()
-	}
-
-	runtimeWrapper.ResetWarmInstanceFunc = func() {
-		runtimeWrapper.runtimeContext.ResetWarmInstance()
-	}
-
```
Comment (camilbancioiu) [inline mock/context/runtimeContextWrapper.go:257]: The fields `IsWarmInstanceFunc` and `ResetWarmInstanceFunc` should be deleted as well from the struct.

**Review by andrei-marinica — DISMISSED** [review]

File: test/features/composability/mandos/forwarder_call_async_accept_esdt.scen.json [hunk 2]
```diff
@@ -61,10 +61,11 @@
...
                         "address": "sc:vault",
-                        "identifier": "str:accept_funds",
-                        "topics": [
-                            "str:TEST-TOKEN",
-                            "str:FungibleESDT",
+    "identifier": "str:accept_funds",
+    "topics": [
+        "str:accept_funds",
+        "str:TEST-TOKEN",
+        "str:FungibleESDT",
```
Comment (camilbancioiu) [inline test/features/composability/mandos/forwarder_call_async_accept_esdt.scen.json]: Indentation is off.

**Review by camilbancioiu — APPROVED** [review]

File: test/features/composability/mandos/forwarder_call_async_accept_esdt.scen.json (hunk 2 above)
Comment (andrei-marinica) [reply test/features/composability/mandos/forwarder_call_async_accept_esdt.scen.json]: I fixed it

Comment (sasurobert) [reply test/features/composability/mandos/forwarder_call_async_accept_esdt.scen.json]: done

File: mock/context/runtimeContextWrapper.go (hunk 1 above)
Comment (sasurobert) [reply mock/context/runtimeContextWrapper.go:257]: done

**Review by camilbancioiu — APPROVED** [review]

**Review by andrei-marinica — APPROVED** [review]

**Pull request merged**